modules/openapi-generator/src/main/resources/typescript/api/api.mustache

```diff
@@ -1,21 +1,21 @@
 // TODO: better import syntax?
-import { BaseAPIRequestFactory, RequiredError } from './baseapi';
-import {Configuration} from '../configuration';
-import { RequestContext, HttpMethod, ResponseContext, HttpFile} from '../http/http';
+import { BaseAPIRequestFactory, RequiredError } from './baseapi{{#platforms}}{{#deno}}.ts{{/deno}}{{/platforms}}';
+import {Configuration} from '../configuration{{#platforms}}{{#deno}}.ts{{/deno}}{{/platforms}}';
+import { RequestContext, HttpMethod, ResponseContext, HttpFile} from '../http/http{{#platforms}}{{#deno}}.ts{{/deno}}{{/platforms}}';
```

Comment:
Adding conditional `.ts` suffixes to all imports seems very verbose to me.
Is it possible to simply add it for all platforms?

Reply:
We may be able to write like any other platform using [Import maps](https://deno.land/manual/linking_to_external_code/import_maps).
But this is unstable feature..

I have another suggestion.

#### TypeScriptClientCodegen.java
```java
if (propPlatform.equals("deno")) {
    additionalProperties.put("extensionForDeno", ".ts");
}
```

#### template(mustache)

as-is
```typescript
import { HttpLibrary } from "./http/http{{#platforms}}{{#deno}}.ts{{/deno}}{{/platforms}}";
```

to-be
```typescript
import { HttpLibrary } from "./http/http{{extensionForDeno}}";
```

Is this acceptable ?

Reply:
That looks better maintainable.
I was thinking whether we can could just always use the `.ts` extension though!?

Reply:
I have fixed above.